ts/features/idpay/initiative/details/components/TransactionDetailsComponent.tsx

```diff
@@ -0,0 +1,167 @@
+import React from "react";
```

Comment:
Couldn't this file use some standard components to share with `RefundDetailsComponent` in order to increase readability and reduce duplication?